README.md

```diff
@@ -1,3 +1,10 @@
+[![Documentation Status](https://readthedocs.org/projects/khiva-python/badge/?version=latest)](https://khiva-python.readthedocs.io/en/latest/?badge=latest)
+[![Build Status](https://travis-ci.com/shapelets/khiva-python.svg?branch=develop)](https://travis-ci.com/shapelets/khiva-python)
+[![codecov.io](https://codecov.io/github/shapelets/khiva-python/coverage.svg?branch=develop)](https://codecov.io/github/shapelets/khiva-python)
```

Comment:
If you want, you can add another for master, look at the following example:
https://github.com/conan-io/conan